chart/open-feature-operator/README.md

```diff
@@ -38,10 +38,39 @@ The command removes all the Kubernetes components associated with the chart and
 ## Configuration
 <a name="configuration"></a>
 
+### Sidecar configuration
 | Value       | Default     | Explanation |
 | ----------- | ----------- | ----------- |
-| `defaultNamespace`      | `open-feature-operator`  | [INTERNAL USE ONLY] To override the namespace use the `--namespace` flag. This default is provided to ensure that the kustomize build charts in `/templates` deploy correctly when no `namespace` is provided via the `-n` flag.|
+| `sidecarConfiguration.envVarPrefix`      | `FLAGD`  | Sets the prefix for all environment variables set in the injected sidecar |
+| `sidecarConfiguration.port`      | 8013  | Sets the value of the `XXX_PORT` environment variable for the injected sidecar container.|
+| `sidecarConfiguration.metricsPort`      | 8014  | Sets the value of the `XXX_METRICS_PORT` environment variable for the injected sidecar container.|
+| `sidecarConfiguration.socketPath`      | `""`  | Sets the value of the `XXX_SOCKET_PATH` environment variable for the injected sidecar container.|
+| `sidecarConfiguration.repository`      | `ghcr.io/open-feature/flagd`  | Sets the image for the injected sidecar container|
+| `sidecarConfiguration.tag`      | current flagd version: `v0.3.0`  | Sets the version tag for the injected sidecar container |
+| `sidecarConfiguration.providerArgs`      | `""`  | Used to append arguments to the sidecar startup command. This value is a comma separated string of key values separated by '=', e.g. `key=value,key2=value2` results in the appending of `--sync-provider-args key=value --sync-provider-args key2=value2` |
 
+### Operator resource configuration
```

Comment:
It might be nice to have an example somewhere of passing one or two of these, just for the uninitiated.

Reply:
example added for port and memory overwrites